common/src/main/webapp/sass/explore.scss

```diff
@@ -25,6 +25,11 @@ $tree-section-width: 300px;
   background: var(--control-background-color);
 }
 
+// Should we just set this in the "lucuma" styles?
+.p-accordion .p-accordion-header:not(.p-disabled) .p-accordion-header-link:focus {
+  box-shadow: none !important;
```

Comment:
We should probably put it in the lucuma styles. Can it be done without the `!important`?

Reply:
I expect the !important not to be needed if it's in the base style. I'll remove from here.